src/components/chat-view-container/chat-view-container.tsx

```diff
@@ -106,12 +109,11 @@ export class Container extends React.Component<Properties, State> {
     if (channelId && channelId !== prevProps.channelId) {
       this.props.stopSyncChannels(prevProps);
       this.props.fetchMessages({ channelId });
-      this.setState({
-        reply: null,
-      });
+      this.setState({ reply: null });
     }
 
     if (channelId && prevProps.user.data === null && this.props.user.data !== null) {
+      this.props.setActiveChannelId(channelId);
```

Comment:
Do we need this any more? We already have the update events in the higher level component so this would just duplicate that, wouldn't it?

Reply:
I actually couldn't reproduce this one (condition), but left it just to make sure i don't break anything. I will remove it